eng/pipelines/runtimelab.yml

```diff
@@ -0,0 +1,220 @@
+# Setting batch to true, triggers one build at a time.
```

Comment:
should we move this template to `eng/pipelines/runtimelab/runtimelab.yml`? so that when we grep for parameters that are only used here (like `dependsOn`), we find them in `runtime` and no-one breaks the `runtimelab` pipeline?

Reply:
not sure I follow the protecting against the break comment but ideally I would like to keep it in the root of pipelines if possible since at least in my mind all the ymls in the root of pipelines represent actual pipelines we have in AzDO. I don't have a strong opinion on this though, so if you still want me to move it I'm fine with that.

Reply:
The thing in question here, is if this yml will be ported to `dotnet/runtime`? I just want it to be there, so that when we grep for yml parameters, or templates usage, we get usages on this template.

Reply:
Yes the idea is to port this guy to dotnet/runtime as well. The reason for that is that we want experiments in runtimelab to only contain the diff of the actual feature being added, as opposed to change in ymls and such other files that would make it harder to maintain or to port back to runtime once the experiment is ready for that.

Reply:
Ok, cool then I'm fine with where it is.